src/commands/key.rs

```diff
@@ -0,0 +1,176 @@
+use aws_sdk_kms::{model::KeySpec, model::KeyUsageType, Client};
+use failure::Error;
+use gtmpl::template;
+use gtmpl_derive::Gtmpl;
+use home;
+use serde::Deserialize;
+use std::env;
+use tokio::process::Command;
+use tracing::instrument;
+use uuid::Uuid;
+
+pub const KEY_POLICY_TEMPLATE: &str = r##"{
```

Comment:
any reason why we're not treating this the same way as the CF Template? seems like we already have all the structure to set things up that way, not sure why we'd do differently

Reply:
That's a cloudformation specific thing as far as I can tell.